### DISCUSSION TITLE
Error imagemagick

### DISCUSSION THREAD

**Discussion by HedgeShot** [nextcloud/all-in-one#2066] [Questions] [answered]
hi, 
I am doing the update to latest and the logs show an error when trying to install imagemagick. I tried to curl those URL from the server and get a valid answer (as long as I put a trailing slash, otherwise I get 301 Moved Permanently):
```
              now              
-------------------------------
 2023-02-24 11:29:41.388097+00
(1 row)

+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
Installing imagemagick via apk...
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.16/main: network error (check Internet connection and firewall)
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.16/community: network error (check Internet connection and firewall)
ERROR: unable to select packages:
The packet imagemagick was not installed!
Enabling Imagick...
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.16/main: network error (check Internet connection and firewall)
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.16/community: network error (check Internet connection and firewall)
ERROR: unable to select packages:
Could not install PHP extension imagick!
```

**Comment by szaimen** [comment, marked as answer]
Hi, can you check if this fixes the issue for you? https://github.com/nextcloud/all-in-one/discussions/2065

**Reply by HedgeShot** [reply]
yes it did the trick. The bug was worst than this, i couldn't reach the nextcloud instance anymore.
Logs still show an error after following the steps listed in your link:
`ERROR [notify_push] src/main.rs:77: Self test failed: Error while communicating with nextcloud instance
`
thanks!
